package.json

```diff
@@ -110,6 +110,8 @@
     "serve": "^11.3.2",
     "start-server-and-test": "^1.11.0",
     "styled-components": "^5.3.0",
+    "styled-system": "^5.1.5",
+    "tslib": "^2.3.0",
```

Comment:
Is tslib required to be installed explicitly? Is it not enough to just set `importHelpers: true` in the tsconfig?

Reply:
You should be able to use tslib through typescript, so you can remove this from package.json, and revert the change to the yarn.lock.